test/e2e/__snapshots__/Iframe.test.js.snap

```diff
@@ -24,7 +24,8 @@ Array [
 
 exports[`Client iframe console.log liveReload disabled 1`] = `
 Array [
-  "Hey.",
+  "Failed to load resource: the server responded with a status of 404 (Not Found)",
+  "Failed to load resource: the server responded with a status of 404 (Not Found)",
```

Comment:
Looks something wrong

Reply:
yes liveReload is disabled and hot is disabled too that's why the WebSocket is not working

Reply:
:+1:

Reply:
I think it is possible that this could be a variable number of 404 error messages, since the client keeps retrying to connect. Maybe this should be handled some way for this particular test?

Reply:
@Loonride any idea to handle these 404 errors?

Reply:
> @Loonride any idea to handle these 404 errors?

@EslamHiko Sorry I missed your reply, it seems tests generally pass as the timing is consistent to only output the error message twice, but just in case:

You could check that every element in that array is the same. Then make the snapshot only test the first element of the output array. This will allow for a variable number of the same error message.